### DISCUSSION TITLE
EddieHub BattleSnake Tournament Results

### DISCUSSION THREAD

**Discussion by SashoStoichkov** [EddieHubCommunity/support#928] [Conference Notes]
Results for the EddieHub BattleSnake Tournament

https://www.youtube.com/watch?v=Aygyo4XI8h0

**Comment by eddiejaoude** [comment]
Winners

1. [x] [DrunkerSnake](https://play.battlesnake.com/u/nitesh-thapliyal/niteshdrunkersnake/) $25
2. [x] [SmolSnake](https://play.battlesnake.com/u/kcoder63/smolsnake/) $10
3. [x] [Cerberus](https://play.battlesnake.com/u/galahad42/cerberus/) $10

Store https://store.battlesnake.com

**Reply by eddiejaoude** [reply]
Video of our winners! https://twitter.com/eddiejaoude/status/1361443550030921735

**Comment by SashoStoichkov** [comment]
"Python" - https://github.com/SashoStoichkov/BattleSnakePython100

Congrats to the Winners: @Nitesh-thapliyal @kcoder63 @galahad42 🎉

**Reply by galahad42** [reply]
Thank you @SashoStoichkov

**Comment by galahad42** [comment]
Cerberus - https://github.com/galahad42/Cerberus    || 3rd prize winner

**Comment by Nitesh-thapliyal** [comment]
Thank you @SashoStoichkov 🙌

**Comment by jai-dewani** [comment]
Mine "BuildingBots" - https://github.com/jai-dewani/battle-snake-bot

**Comment by Nitesh-thapliyal** [comment]
can I share the Battlesnake code after the spring league 2021, I need to work on it :sweat_smile:

**Reply by jai-dewani** [reply]
Secret Sause 🔐  
Best of luck for Spring League
